src/native/minipal/minipalconfig.h.in

```diff
@@ -1,7 +1,9 @@
 #ifndef HAVE_MINIPAL_MINIPALCONFIG_H
 #define HAVE_MINIPAL_MINIPALCONFIG_H
 
+#cmakedefine01 HAVE_ARC4RANDOM_BUF
 #cmakedefine01 HAVE_AUXV_HWCAP_H
+#cmakedefine01 HAVE_O_CLOEXEC
```

Comment:
[minipal/configure.cmake](https://github.com/dotnet/runtime/blob/main/src/native/minipal/configure.cmake) also needs to be updated to set the `HAVE_ARC4RANDOM_BUF` and `HAVE_O_CLOEXEC` variables.

Reply:
This is how libs is checking:

https://github.com/dotnet/runtime/blob/a4a8e8e712eff30c9223ebdeeeb4c80fb798e972/src/native/libs/configure.cmake#L257-L260

https://github.com/dotnet/runtime/blob/a4a8e8e712eff30c9223ebdeeeb4c80fb798e972/src/native/libs/configure.cmake#L124-L127

It also looks like the random functions are the only ones using `HAVE_ARC4RANDOM_BUF`, so that can be removed from the libs configure.cmake/pal_config.h.in.